integration-tests/features/label-information/get/name-match.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Get label information by specifying the name.
+
+    Background:
+        * url apiHost
+
+    Scenario Outline: Validate the query results when matching against name.
+
+        Given path 'label-lookup', name
+        When method get
+        Then assert responseStatus == exStatus
+        And match response == read( expected )
+
+        Examples:
+            | exStatus | name                      | expected                |
+            | 200      | health-services-research  | name-match-exact-match.json   |
+            | 404      | chicken                   | name-match-not-a-match.json   |
```

Comment:
Please add explicit tests for:

* label string matches. (e.g. `basic_science`)
* pretty-url string matches (e.g. `health-services-research`)
* both strings match (e.g. `treatment`)

The names for the 'expected' files should reflect the case. (e.g. `name-match-id-string-matches.json`). 

The feature file's `Then` clause then becomes `Then status 200` and the "match-not-found" cases move to their own `.feature` file.

The feature you copied from only has two scenarios right now, and that's going to change with the new design.